Fix install locations in the 2.15 Makefile: the binary was installed
into MANDIR instead of BINDIR, and uninstall ignored DESTDIR.

Index: bs-2.15/Makefile
===================================================================
--- bs-2.15.orig/Makefile
+++ bs-2.15/Makefile
@@ -53,9 +53,9 @@ install: bs.6 uninstall
 	install -m 0755 -d $(DESTDIR)/usr/share/icons/hicolor/32x32/apps/
 	install -m 0755 -d $(DESTDIR)/usr/share/appdata
-	install -m 0755 bs $(DESTDIR)$(MANDIR)/
+	install -m 0755 bs $(DESTDIR)$(BINDIR)/
 	install -m 0644 bs.6 $(DESTDIR)$(MANDIR)/man6/
 	install -m 0644 bs.desktop $(DESTDIR)/usr/share/applications/
 	install -m 0644 battleship.png $(DESTDIR)/usr/share/icons/hicolor/32x32/apps/
 	install -m 0644 bs.adoc $(DESTDIR)/usr/share/appdata/

 uninstall:
-	rm -f $(BINDIR)/bs /usr/share/man/man6/bs.6
+	rm -f $(DESTDIR)$(BINDIR)/bs $(DESTDIR)/usr/share/man/man6/bs.6

 # Export
